src/pages/Staydetail/Facilities/Facilities.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  FaSwimmingPool,
+  FaHotTub,
+  FaHouseUser,
+  FaFireAlt,
+  FaParking,
+  FaWifi,
+} from 'react-icons/fa';
+
+const Facilities = ({ services, amenities }) => {
+  return (
+    <Section>
+      <RoomService>숙소 편의시설</RoomService>
+      <ListBox>
+        <SmallSection>
+          <Icon>
+            <FaSwimmingPool />
+          </Icon>
+          {services && services[0]}
+        </SmallSection>
+        <Wifi>
+          <SmallSection>
+            <Icon>
+              <FaWifi />
+            </Icon>
+            {amenities && amenities[0]}
+          </SmallSection>
+        </Wifi>
+      </ListBox>
+      <ListBox>
+        <SmallSection>
+          <Icon>
+            <FaHotTub />
+          </Icon>
+          {services && services[1]}
+        </SmallSection>
+        <Kitchen>
+          <SmallSection>
+            <Icon>
+              <FaFireAlt />
+            </Icon>
+            {amenities && amenities[1]}
+          </SmallSection>
+        </Kitchen>
+      </ListBox>
+      <ListBox>
+        <SmallSection>
+          <Icon>
+            <FaHouseUser />
+          </Icon>
+          {services && services[2]}
+        </SmallSection>
+        <Parking>
+          <SmallSection>
+            <Icon>
+              <FaParking />
+            </Icon>
+            {amenities && amenities[2]}
+          </SmallSection>
+        </Parking>
+      </ListBox>
+    </Section>
+  );
+};
+
+export default Facilities;
+
+const Section = styled.div`
+  border: none;
+  border-bottom: solid 1px #bfcace;
+  margin-top: 10px;
+`;
+
+const RoomService = styled.div`
+  font-size: 22px;
+  padding-bottom: 30px;
+`;
+
+const ListBox = styled.div`
+  display: flex;
+  width: 400px;
+`;
+
+const SmallSection = styled.div`
+  ${props => props.theme.flexBox('row', 'center', 'flex-start')}
```

Comment:
row는 flex의 기본 속성이기 때문이기 때문에 빈 스티링으로 두어도 됩니다!
그리고, 구조 분해 할당을 할 수 있습니다.
```suggestion
  ${({theme}) => theme.flexBox('', 'center', 'flex-start')}
```

Reply:
반영하였습니다!